### ISSUE TITLE
direnv: shell.nix doesn't exist

### ISSUE THREAD

**Issue by doronbehar** [makefu/hydra-check#1]
Hey, just wanted to checkout out this after reading https://discourse.nixos.org/t/hydra-check-does-package-x-still-build-on-channel-y/6126/1 and noticed that `direnv allow` fails with:

```
direnv: loading /var/src/hydra-check/.envrc
direnv: cannot watch file shell.nix because it does not exist
direnv: error exit status 1
```

**Comment by makefu** [comment]
good catch, i seem to have forgotten it. you can however still run `nix-shell` and it will catch up the existing `default.nix`

**Comment by makefu** [comment]
should be fixed with 9680e78e9abd135f5b0f668e919aceaa339d1e0a
